components/CopyId.tsx

```diff
@@ -6,7 +6,11 @@ import useClipboard from '../lib/hooks/useClipboard';
 
 import { Tooltip, TooltipContent, TooltipTrigger } from './ui/Tooltip';
 
-export const CopyID = ({ children, tooltipLabel = <FormattedMessage defaultMessage="Copy ID" id="wtLjP6" /> }) => {
+export const CopyID = ({
+  children,
+  prepend = '',
```

Comment:
It would probably be more straightforward to inform a `value` property you want to be copied and always display children as the label.

Reply:
Good suggestion, pushed a fix for this!